snap/snapcraft.yaml

```diff
@@ -52,7 +52,7 @@ parts:
   ros2cli:
     plugin: colcon
     source: https://github.com/ros2/ros2cli.git
-    source-branch: jazzy
+    source-tag: 0.32.1
```

Comment:
isn't latest jazzy tag, 0.35?

Reply:
On the jazzy branch the latest release is 0.32.1. Are you looking at rolling?

Reply:
yes, rolling is 0.33+, jazzy is 0.32.N